tame.lua

```diff
@@ -0,0 +1,38 @@
+-- tame and train animals {set animals as Wild through to Masterfully Trained}
+
+local help=[====[
+tame
+====
+Tame and train animals.
+
+Usage: tame -set <level>
+
+:0: wild
+:1: trained
+:2: well-trained
+:3: skillfully trained
+:4: expertly trained
+:5: exceptionally trained
+:6: masterfully trained
+:7: tame
+:8: semi-wild
+]====]
+
+local utils = require('utils')
+local selected = dfhack.gui.getSelectedUnit()
+local validArgs = utils.invert({'set'})
+local args = utils.processArgs({...}, validArgs)
+--[

```

Comment:
Might want to remove these?

Reply:
Presumably you mean the defunct comments from lines 25 and 37